.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "tabWidth": 2,
```

```suggestion
  "tabWidth": 4,
```
Is 4 okay to you? Előd and I certainly prefer it.